files/scripts/supervisor-proc-exit-listener

```diff
@@ -77,7 +78,8 @@ def generate_alerting_message(process_name):
     else:
         namespace = namespace_prefix + namespace_id
 
-    syslog.syslog(syslog.LOG_ERR, "Process '{}' is not running in namespace '{}'.".format(process_name, namespace))
+    syslog.syslog(syslog.LOG_ERR, "Process '{}' is not running in namespace '{}'({} minutes)."
```

Comment:
Add a space before the opening parenthesis. E.g., `"Process '{}' is not running in namespace '{}' ({} minutes)."`

Reply:
Fixed.